tests/standalone_tests/test_monetdb_interface_if_not_exists.py

```diff
@@ -0,0 +1,184 @@
+    )
+    localnode1_db_cursor.execute(
+        f"CREATE TABLE {info.name} ( {info.column_names[0]} {info.schema_.columns[0].dtype.value})"
+    )
+    localnode1_db_cursor.execute(
+        f"INSERT INTO {info.name} ( {info.column_names[0]} ) VALUES ( 10 )"
+    )
+    return info
+
+
+def test_create_view_if_not_exists(
+    use_localnode1_database,
+    create_sample_table,
+):
+    table_info = create_sample_table
+    view_name = "view_test_create_view_if_not_exists"
+
+    create_view(
+        view_name=view_name,
+        table_name=table_info.name,
+        columns=table_info.column_names,
+        filters=None,
+        minimum_row_count=0,
+        check_min_rows=False,
+    )
+
+    try:
+        create_view(
+            view_name=view_name,
+            table_name=table_info.name,
+            columns=table_info.column_names,
+            filters=None,
+            minimum_row_count=0,
+            check_min_rows=False,
+        )
+    except Exception as exc:
+        pytest.fail(f"A view should be able to be recreated. Exception: \n{str(exc)} ")
+
+
+def test_create_table_if_not_exists(
+    use_localnode1_database,
+):
+    table_name = "test_create_table_if_not_exists"
+    table_schema = TableSchema(
+        columns=[ColumnInfo(name="sample_column", dtype=DType.INT)]
+    )
+    create_table(
+        table_name=table_name,
+        table_schema=table_schema,
+    )
+
+    try:
+        create_table(
+            table_name=table_name,
+            table_schema=table_schema,
+        )
+    except Exception as exc:
+        pytest.fail(f"A table should be able to be recreated. Exception: \n{str(exc)} ")
+
+
+def test_create_remote_table_if_not_exists(
+    use_localnode1_database,
+):
+    table_name = "test_create_remote_table_if_not_exists"
+    table_schema = TableSchema(
+        columns=[ColumnInfo(name="sample_column", dtype=DType.INT)]
+    )
+    create_remote_table(
+        name=table_name,
+        monetdb_socket_address="127.0.0.1:50000",
+        schema=table_schema,
+    )
+
+    try:
+        create_remote_table(
+            name=table_name,
+            monetdb_socket_address="127.0.0.1:50000",
+            schema=table_schema,
+        )
+    except Exception as exc:
+        pytest.fail(
+            f"A remote table should be able to be recreated. Exception: \n{str(exc)} "
+        )
+
+
+def test_create_merge_table_if_not_exists(
+    use_localnode1_database,
+):
+    table_name = "test_create_merge_table_if_not_exists"
+    table_schema = TableSchema(
+        columns=[ColumnInfo(name="sample_column", dtype=DType.INT)]
+    )
+    create_merge_table(
+        table_name=table_name,
+        table_schema=table_schema,
+    )
+
+    try:
+        create_merge_table(
+            table_name=table_name,
+            table_schema=table_schema,
+        )
+    except Exception as exc:
+        pytest.fail(
+            f"A merge table should be able to be recreated. Exception: \n{str(exc)} "
+        )
```

Comment:
The SQL standard defines the behavior of the `IF NOT EXISTS` clause and MonetDB [confronts to the standard](https://www.monetdb.org/documentation-Sep2022/user-guide/sql-manual/data-definition/table-definition/). 

If I am not mistaken, here you are testing that MonetDB actually confronts to the standard. Why not just test our code by having unit tests validate that the correct query is produced?

Reply:
Added unit tests as well testing all the scenarios.

I prefer to leave the integrations tests as well to make sure that the test case as a "whole" is working, the lower stack could potentially change but the test case should work whatever the change.